### PR TITLE
Authentication Pages 

### PR DESCRIPTION
**Pull request by saiganesharaja**
Screenshot from 2023-12-26 10-51-44
Screenshot from 2023-12-26 10-52-52

### CODE REVIEW

File: src/app/signin/page.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+'use client';
+import React from 'react';
+import { LockOutlined, UserOutlined } from '@ant-design/icons';
+import { Button, Checkbox, Form, Input } from 'antd';
+import Link from 'next/link';
+
+
+const App: React.FC = () => {
```
Comment (NagaDurga16) [inline src/app/signin/page.tsx]: change function name to signin

Comment (NagaDurga16) [reply src/app/signin/page.tsx]: please change name

Comment (tulasikurete) [inline src/app/signin/page.tsx]: change function name

File: src/app/signup/page.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+'use client';
+import React from 'react';
+import { LineHeightOutlined, LockOutlined, UserOutlined } from '@ant-design/icons';
+import { Button, Checkbox, Form, Input } from 'antd';
+import Link from 'next/link';
+
+const App: React.FC = () => {
```
Comment (NagaDurga16) [inline src/app/signup/page.tsx]: please change function name

**Comment by NagaDurga16** [comment]
Please remove rating file in authentication branch

**Review by tulasikurete — APPROVED** [review]

**Review by NagaDurga16 — APPROVED** [review]

File: src/app/cart/page.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 "use client";
 
 import { getCartItems, updateCartItemQuantity } from "@/lib/api";
-import { CartItem } from "@prisma/client";
+
```
Comment (arajajyothibabu) [inline src/app/cart/page.tsx:4]: Why cartitem removed here?

File: src/app/page.tsx [hunk 4]
```diff
@@ -2,8 +2,9 @@
 
 import { addCartItem, getItems } from "@/lib/api";
 import { Item } from "@prisma/client";
-import { Card, List, Space, Typography, Image, Button, theme } from "antd";
-import { useEffect, useState } from "react";
+import { Card, List, Space, Typography, Image, Button, theme, App } from "antd";
+import { Component, useEffect, useState } from "react";
+import Rating from "./components/Rating";
```
Comment (arajajyothibabu) [inline src/app/page.tsx]: Why Rating came here ?

File: src/app/page.tsx [hunk 5]
```diff
@@ -65,7 +66,9 @@ export default function Home() {
             </Card>
           </List.Item>
         )}
-      />
+      /> */}
+      <Rating/>
```
Comment (arajajyothibabu) [inline src/app/page.tsx]: Remove rating changes from this branch

**Review by arajajyothibabu — CHANGES_REQUESTED** [review]

**Review by NagaDurga16 — CHANGES_REQUESTED** [review]
remove rating page

**Review by NagaDurga16 — APPROVED** [review]
remove rating page

**Comment by saiganesharaja** [comment]
Screenshot from 2024-01-03 11-43-01

**Review by tulasikurete — APPROVED** [review]

**Review by arajajyothibabu — APPROVED** [review]

**Pull request merged**